TypeCobol.Test/Parser/Programs/TypeCobol/GlobalStorageSection.rdzMix.txt

```diff
@@ -83,6 +84,7 @@ Line 58[8,30] <27, Error, Syntax> - Syntax error : GLOBAL-STORAGE SECTION is onl
        PROCEDURE DIVISION.
       *G-Date and G-TechCtx are accessible 
       * in nested procedure from Global Storage
```

Comment:
same remark as above

Reply:
Test modified by using IN-OUT
https://github.com/TypeCobolTeam/TypeCobol/tree/55e817f33a3d27d5c35869ccddd44cd00f20b192